SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -701,7 +779,9 @@ Worker_CommandRequest USpatialSender::CreateRPCCommandRequest(UObject* TargetObj
 		}
 	}
 
-	WriteRpcPayload(RequestObject, TargetObjectRef.Offset, CommandIndex, PayloadWriter);
+	TArray<uint8> Data(PayloadWriter.GetData(), PayloadWriter.GetNumBytes());
```

Comment:
Can you make PayloadWriter.GetData() and PayloadWriter.GetNumBytes() as a part of the constructor for the RPCPayload so we don't have this extra line?